packages/core/src/cart/errors/cart-consistency-error.ts

```diff
@@ -0,0 +1,17 @@
+import { StandardError } from '../../common/error/errors';
+
+/**
+ * This error is thrown when the server detects inconsistency in cart data since it is last requested,
+ * for example, product prices or eligible discounts have changed.
+ */
+export default class CartConsistencyError extends StandardError {
+    constructor(message?: string) {
+        super(
+            message ||
+                'Your checkout could not be processed because some details have changed. Please review your order and try again.',
```

Comment:
I forgot if we translate these messages or not? Or if there was a mechanism to do the translation since this is a user facing message.

Reply:
@animesh1987 It gets translated in `checkout-js`. See  https://github.com/bigcommerce/checkout-js/pull/1138/files#diff-00368bf808596bf313d17f20d0b174a28cd10bfb3e6651656cec53e395016f6aR29